.trivyignore

```diff
@@ -37,3 +37,4 @@ CVE-2020-8559
 # https://lists.debian.org/debian-lts-announce/2024/11/msg00030.html
 DLA-3972-1
 DLA-4085-1
+DLA-4105-1
```

Comment:
We've been ignoring these timezone related trivy failures. See the comment above for the explanation.